### PR TITLE
Issue 701 - add deployment version, copy OEP markdowns

### LINKED ISSUE #701: Add deployment version, copy OEP markdowns
**Is your feature request related to a problem? Please describe.**
To do:
- [x] Move READMEs from https://github.com/oceanprotocol/oeps to docs website
- [x] Create Documentation related to upcoming v4 + NFT (explain NFTs in general and show Ocean Protocol will support those)
- [x] Create a Deployments page to show in which network what version of contracts, Aquarius, Provider, etc is running.

### PR DESCRIPTION
**Pull request by akshay-ap**
Fixes #701 

Changes proposed in this PR:

- Add Deployment version page
- Copy OEPs markdowns: Asset DDO and DID

### CODE REVIEW

**Comment by akshay-ap** [comment]
image


image

File: src/components/Deployments.jsx [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+import PropTypes from 'prop-types'
+import { graphql } from 'gatsby'
+
+export default function Deployments({ data, location }) {
+  const [content, setContent] = useState(undefined)
+  const [loading, setLoading] = useState(true)
+
+  const networks = {
+    'Ethereum Mainnet': {
+      aquarius: 'https://aquarius.mainnet.oceanprotocol.com',
```
Comment (alexcos20) [inline src/components/Deployments.jsx:18]:
```suggestion
      aquarius: 'https://aquarius.oceanprotocol.com',
```

File: src/components/Deployments.jsx [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+      provider: 'https://provider.mainnet.oceanprotocol.com'
+    },
+    'Polygon Mainnet': {
+      aquarius: 'https://aquarius.polygon.oceanprotocol.com',
```
Comment (alexcos20) [inline src/components/Deployments.jsx:22]:
```suggestion
      aquarius: 'https://aquarius.oceanprotocol.com',
```

File: src/components/Deployments.jsx [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+      provider: 'https://provider.polygon.oceanprotocol.com'
+    },
+    'Binance Smart Chain': {
+      aquarius: 'https://aquarius.bsc.oceanprotocol.com',
```
Comment (alexcos20) [inline src/components/Deployments.jsx:26]:
```suggestion
      aquarius: 'https://aquarius.oceanprotocol.com/',
```

File: src/components/Deployments.jsx [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+      provider: 'https://provider.bsc.oceanprotocol.com'
+    },
+    Ropsten: {
+      aquarius: 'https://aquarius.ropsten.oceanprotocol.com',
```
Comment (alexcos20) [inline src/components/Deployments.jsx:30]:
```suggestion
      aquarius: 'https://aquarius.oceanprotocol.com',
```

File: src/components/Deployments.jsx [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+      provider: 'https://provider.ropsten.oceanprotocol.com'
+    },
+    Rinkeby: {
+      aquarius: 'https://aquarius.rinkeby.oceanprotocol.com',
```
Comment (alexcos20) [inline src/components/Deployments.jsx:34]:
```suggestion
      aquarius: 'https://aquarius.oceanprotocol.com',
```

File: src/components/Deployments.jsx [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+      provider: 'https://provider.rinkeby.oceanprotocol.com'
+    },
+    Mumbai: {
+      aquarius: 'https://aquarius.mumbai.oceanprotocol.com',
```
Comment (alexcos20) [inline src/components/Deployments.jsx:38]:
```suggestion
      aquarius: 'https://aquarius.oceanprotocol.com',
```

**Review by alexcos20 — CHANGES_REQUESTED** [review]
we switched to multi network aqua, so it's the same instance for all networks

**Comment by alexcos20** [comment]
beside my suggestions, looks awesome

**Review by alexcos20 — APPROVED** [review]

**Pull request merged**